### DISCUSSION TITLE
Task Planing in Bulk

### DISCUSSION THREAD

**Discussion by cadev71** [madrecha/jamku#1243] [QA]
1. It will be Easer if plan to Task can be added in Bulk, for different staffs at a time
2. Whiling adding plan task if can be done with Task List with services can be Easer
3. If stages or Todoes can be added to plan 
4.  Task planning if can be imported with excel can ease our effort specially at the Return filling time

**Comment by adarshmadrecha** [comment]
Nice suggestions. Creating issue to track this feature request.
https://github.com/madrecha/jamku/issues/1245
